src/calculateAverage.js

```diff
@@ -37,7 +37,35 @@ function applyCalculateAverage() {
    * @param {Object} options - optional
    */
   [].__proto__.calculateAverage = function(options) {
-    // write code here
+    if (options === undefined) {
+      return this.reduce((acum, num) => acum + num, 0) / this.length;
+    }
+
+    if (options === null) {
+      return undefined;
```

Comment:
не потрібно явно вказувати `undefined`
```suggestion
      return;
```